core/modules/views_ui/css/views_ui.admin.theme.css

```diff
@@ -217,6 +217,45 @@ input.form-radio {
   margin-top: 6px;
 }
 
+.form-item-show-wizard-key,
+.form-item-show-type,
+.form-item-show-tagged-with,
+.form-item-show-sort {
+  display: inline-block;
+  white-space: nowrap;
+  padding: 0 0 0.5em;
+}
+
+@media (max-width: 600px) {
+  [id="edit-block-style"] div,
+  [id="edit-block-style"] label,
+  [id="edit-block-style"] input,
+  [id="edit-page-style"] div,
+  [id="edit-page-style"] label,
+  [id="edit-page-style"] input {
+    display: inline-block;
```

Comment:
This is just for the 'Add view' wizard, so I don't think other displays are an issue here, but should probably get a class for these anyway, and the attribute selector is pretty dumb.